src/iris/bin/sync_targets.py

```diff
@@ -304,7 +304,7 @@ def get_ldap_lists(l, search_strings, parent_list=None):
                              serverctrls=[req_ctrl],
                              attrlist=(search_strings['list_cn_field'], search_strings['list_name_field']),
                              filterstr=filterstr)
-        rtype, rdata, rmsgid, serverctrls = l.result3(msgid, resp_ctrl_classes=known_ldap_resp_ctrls)
+        rtype, rdata, rmsgid, serverctrls = l.result3(msgid, timeout=20, resp_ctrl_classes=known_ldap_resp_ctrls)
```

Comment:
Can we make both instances of this 20 configurable somewhere or a constant at the top of the file?